xbmc/pvr/filesystem/PVRGUIDirectory.cpp

```diff
@@ -205,6 +205,7 @@ void GetSubDirectories(const CPVRRecordingsPath& recParentPath,
       item->SetProperty("totalepisodes", 0);
       item->SetProperty("watchedepisodes", 0);
       item->SetProperty("unwatchedepisodes", 0);
+      item->SetProperty("sizeginbytes", 0.0f);
```

Comment:
typo -> "sizeinbytes" (a 'g' slipped in)

Reply:
Also seen it set a float `0.0f` (32bit)
There is a `CVariant(int64_t integer);` maybe use then as `0LL`.